### ISSUE TITLE
reserve1, elevator puzzle

### ISSUE THREAD

**Issue by mpeltomaa** [loiste-interactive/infra-issues#48]
If you run thru door (to out) game either stops and stays jammed or with luck will continue after 15 - 20 seconds. While game is jammed (actually screen) you can still walk and hear walking sounds. Below is picture of "jam point" :)

![door](https://cloud.githubusercontent.com/assets/6035374/10870116/2f96f91c-80cb-11e5-9e8c-cd4d452fc2cc.jpg)

**Comment by mpeltomaa** [comment]
Seems that this can be done only once per game session, so it is related to scene loading?

**Comment by aleksijuvani** [comment]
There shouldn't be any loading happening here. Do you have an anti-virus or any other software running in the background that could interfere with the game? Does this still happen if you restart the game and go back to the same spot?

**Comment by mpeltomaa** [comment]
No, AV or other stuff. Plain Win 10, display drivers + steam. It will happen again if I reload game. Also if I walk thru door, it goes smoothly and after that running won't make game jam.

**Comment by mpeltomaa** [comment]
I have made more tests on this. On my first gameplay i didnt notice this bug, that time i walked around inspecting different things. Second time i already knew what is where and what to do - so i ran without walking (way before this room) and went straight to outside (to get that thing..) and then it happened. I reproduced this bug couple times (started level again) and ran again with same bug. Last time i walked to that room and then ran outside - it didn't happen.

**Comment by CorruptComputer** [comment]
Does this still happen? I haven't been able to recreate it.
